src/Sql/Sql/Sql.csproj

```diff
@@ -21,12 +21,12 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.48.0-preview" />
+    <Reference Include="D:\src\azure\azure-sdk-for-net\artifacts\bin\Microsoft.Azure.Management.Sql\Debug\netstandard2.0\Microsoft.Azure.Management.Sql.dll" /> 
```

Comment:
This dll can't be found on our test machines if you use absolute path here. Could you tell me why you need specify the path of dll file?